devnet-sdk/devstack/stack/match/interop.go

```diff
@@ -0,0 +1,9 @@
+package match
+
+import "github.com/ethereum-optimism/optimism/devnet-sdk/devstack/stack"
+
+// L2ChainA is an alias for the first L2 network.
+var L2ChainA = First[stack.L2NetworkID, stack.L2Network]()
+
+// L2ChainB is an alias for the second L2 network.
+var L2ChainB = Second[stack.L2NetworkID, stack.L2Network]()
```

Comment:
The naming of these makes them feel a bit too much like magic values. I know in interop tests we've referred to these as ChainA and ChainB a lot and I think that makes sense in the interop dsl struct, but I think here I'd rename to be more generic selectors like `FirstL2Chain` and `SecondL2Chain`. Not a strongly held opinion though.

Reply:
I like them as aliases, I don't count chains, but think of them more as named things. So I'm keeping the aliases for now.